sbncode/CAFMaker/FillReco.cxx

```diff
@@ -930,6 +930,22 @@ namespace caf
 
   //......................................................................
 
+  void FillPFPRazzled(const art::Ptr<sbn::MVAPID> razzled,
+                      caf::SRPFP& srpfp,
+                      bool allowEmpty)
+  {
+    srpfp.razzled.electronScore = razzled->mvaScoreMap.at(11);
+    srpfp.razzled.muonScore = razzled->mvaScoreMap.at(13);
+    srpfp.razzled.photonScore = razzled->mvaScoreMap.at(22);
+    srpfp.razzled.pionScore = razzled->mvaScoreMap.at(211);
+    srpfp.razzled.protonScore = razzled->mvaScoreMap.at(2212);
+
```

Comment:
any particular reason to not include the `otherScore`? I suppose that if the `otherScore` is the best, then the pdg will be -5 and the score will be available in `bestScore` so one doesn't really need that information 😛

Reply:
I personally do not use `otherScore` in my selection but I don't see why not. 
Was wondering if it'll be confusing for analyzer.
Maybe can add a note somewhere in the code explaining the score

Reply:
Thanks for bringing this up @lynnt20 , I hadn't noticed I had left lingering references to 'other' in the module.

The version of Razzled the three of us have all been using doesn't actually have an other score, I will push a commit that removes all references to it. 

The approach was modelled on Razzle & Dazzle which _did_ have `otherScore` categories because that could be filled with 'mis-labelled' particles (muons, protons & pions in Razzle and electrons & photons in Dazzle). So the first iteration of Razzled also contained it but I quickly realised that it wasn't really feasible because it would only contain "exotic" particles like kaons etc and there just weren't enough to train on.